### ISSUE TITLE
Footer and main content missing space

### ISSUE THREAD

**Issue by flora8984461** [EclipseFdn/solstice-assets#185]
This can cause something like: https://projects.eclipse.org/projects/ecd

![image](https://user-images.githubusercontent.com/39588094/97615443-b1754f00-19f1-11eb-9431-554d9006373c.png)

Or something like: https://www.eclipse.org/projects/project_activity.php

![image](https://user-images.githubusercontent.com/39588094/97615549-d4076800-19f1-11eb-888c-ec0ef1fb4c51.png)

**Comment by flora8984461** [comment]
But need to be careful with this, since on Hugo sites, there are some styles have `margin-top` for footer, and we cannot directly add spacing to featured_footer since we have some custom_footer need to be directly connected with featured_footer.

**Comment by chrisguindon** [comment]
I don't think the problem is with the footer. I would expect the .div containing the charts to have a martin-bottom.

I think we want to avoid adding a margin-top to the featured footer because we don't always need it.

For example,
https://sparkplug.eclipse.org/

**Comment by flora8984461** [comment]
> I don't think the problem is with the footer. I would expect the .div containing the charts to have a martin-bottom.
> 
> I think we want to avoid adding a margin-top to the featured footer because we don't always need it.
> 
> For example,
> https://sparkplug.eclipse.org/

Yeah, I think the `region region-content solstice-region-element-count-1` or the `maint-content-area` probably need a margin-bottom at this type of page. But I think we probably add a specific new class name to these pages so that it won't affect others with the same id or class name. 🤔

**Comment by flora8984461** [comment]
Can we add the existing class "margin-bottom-n" to this type of pages https://projects.eclipse.org/projects/[PROJECTNAME] ? 🤔

**Comment by chrisguindon** [comment]
yes, that might be the solution!

**Comment by chrisguindon** [comment]
I need to finish updating our drupal stack so that you can help us update our drupal sites :)

**Comment by ericpoirier** [comment]
I created a patch for the Solstice drupal theme to add the margin-bottom-20 class.
